src/components/table/TableCellExpander.tsx

```diff
@@ -0,0 +1,48 @@
+import { CellProps } from 'react-table';
+import { ReactComponent as IconChevronDown } from 'assets/icons/chevronDown.svg';
+import { PropsWithChildren } from 'react';
+
+interface Props {
+  isExpandedContent?: JSX.Element;
```

Comment:
I'd consider passing functions that return elements - and not directly passing elements. It will be easier for tests, and it's more flexible and powerful because you are within a function. Also - you are avoiding rendering something that might not be used.

Reply:
fixed